### ISSUE TITLE
Logs not created properly and not sync ed. No known reason

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1219]
In GitLab by @franra9 on Jan 30, 2024, 10:10

Hello @dbeltrankyl and @kinow,

### Autosubmit Version

4.1-beta
### Expid affected(If applicable)

not relevant

### Which task has issues? Where is the log(If applicable)

DN

### Summary

Apparently, randomly the logs are not created properly and I end up with logs that are `*out.0` and `*err.0`

These files are not sync to the virtual machine, so the workflow fails as it thinks that the job failed (I think).

### Steps to reproduce

Ghost error, I can not reproduce.

### What is the current bug behavior?

see summary

### What is the expected correct behavior?

logs have the expected time (`.yyyymmddhhss` instead of `.0`) and sync to the vm.

### Relevant logs and/or screenshots(if applicable)

in the vm:

```
856 of 874 jobs remaining (19:15)
Checking 1 jobs for platform LUMI-LOGIN
Job a0it_19900101_fc0_1_2_DN is RUNNING
Checking 1 jobs for platform LUMI
[ERROR] Job a0it_19900101_fc0_1_SIM completion check failed. There is no COMPLETED file[eCode=6009]
No more jobs to run.
Some jobs have failed and reached maximum retrials
```
in the remote:

```
-rw-rw---- 1 francerou project_465000454    5856 Jan 24 19:16 a0it_19900101_fc0_1_2_DN.cmd.out.0
-rw-rw---- 1 francerou project_465000454      22 Jan 24 19:16 a0it_19900101_fc0_1_2_DN_STAT
-rw-rw---- 1 francerou project_465000454       0 Jan 24 19:16 a0it_19900101_fc0_1_2_DN_COMPLETED
-rw-rw---- 1 francerou project_465000454   23376 Jan 24 19:16 a0it_19900101_fc0_1_2_DN.cmd.err.0
```

### Any other relevant information (if applicable)

I talked some days ago about this issue with @kinow via slack with.

()

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jan 30, 2024, 10:23

There is a rework of the logs that I have to finish, I expect that similar issues will be solved when it is ready, it also happens sometimes in 4.0

**Comment by kinow** [comment]
marked this issue as related to #1217

**Comment by kinow** [comment]
See issue linked above for another report of - possibly - this same issue, by @nunomrm 

>In the **a0jp** and **a0js** experiments that I ran, I noticed the logs of the simulation chunks ("SIM") are not being written in the VM (`tmp/LOG_a0js` for example). I didn't use the autosubmit wrapper in a while, and I wonder if it is a new policy made to save space in the VM (because of [the need to manage the space in the VM](https://earth.bsc.es/gitlab/digital-twins/de_340/project_management/-/issues/584)) or if it is a new bug by using the wrapper. Nevertheless, I have access to the logs through LUMI (`/scratch/project_465000454/rochamon/a0jp/LOG_a0jp`).

**Comment by kinow** [comment]
mentioned in issue digital-twins/de_340/project_management#632

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Apr 18, 2024, 08:37

I fixed many issues with logs, specifically with wrappers, during !409. This should be fixed in v4.4

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jun 11, 2024, 11:14

It should be fixed from >4.1.4. If not please, open a new issue and tag this one
